spec/ic-interface-spec.md

```diff
@@ -757,7 +755,7 @@ Rejection codes are member of the following enumeration:
 
 -   `CANISTER_REJECT` (4): Explicit reject by the canister.
 
--   []{#CANISTER_ERROR}`CANISTER_ERROR` (5): Canister error (e.g., trap, no response)
+-   `CANISTER_ERROR` (5): Canister error (e.g., trap, no response)
```

Comment:
This loses the anchor, probably causing a dead link elsewhere. Maybe anchors that are not tied to sections are a bad idea anyways.